src/cryptmd5.ts

```diff
@@ -66,6 +66,17 @@ export class Hash {
     constructor(public value: string) { }
 }
 
+
+export function janus_hash(password: string, salt: string): Hash {
+    if('' === password) {
+        return new Hash('');
+    } else {
+        return crypt_md5(password, salt);
+    }
+}
+
+
+
 /**
```

Comment:
I don't want it on the call-site. I think it belongs to sds because it's a janus-feature. But I could move it to sds.ts.